README.md

```diff
@@ -39,7 +39,28 @@ Please file an issue if additional packages are required.
 * python3.9
 * python3.9-venv
 * musl-1.2.2
-* ARM GCC compiler; 10.2-2020.11
+* cmake
+* ninja-build
+* ARM GCC compiler for none-elf; version 12.2.1 20221205
+* device tree compiler
+* xmllint
+
+To build the documentation you also need
+* pandoc
+* pdflatex
+* texlive-latex-recommended
+* texlive-fonts-recommended
+* texlive-fonts-extra
+* texlive-latex-extra
+
+On a Debian-like system you can do:
+
+    $ sudo apt install build-essential git cmake ninja-build \
+	    device-tree-compiler libxml2-utils \
+		pandoc texlive-latex-base texlive-latex-recommended \
```

Comment:
`texlive-latex-recommended` is not needed

Reply:
actually I might be wrong here - the CI doesn't install it but it might have it already available. I'll check on a fresh install.

Reply:
I installed it to get xcolor.sty amongst other things